Java/README.md

```diff
@@ -0,0 +1,169 @@
+# Random Cut Forest
+
+This directory contains a Java implementation of the Random Cut Forest data structure and algorithms
+for anomaly detection, density estimation, imputation, and forecast. The goal of this library 
+is to be easy to use and to strike a balance between efficiency and extensibility.
+
+## Basic operations
+
+To create a RandomCutForest instance with all parameters set to defaults:
+
+```java
+int dimensions = 1; // The number of dimensions in the input data, required
```

Comment:
Can we make this 5 as an example different from 1?

Reply:
Fixed in new commit.